crypto/fipsmodule/ec/p384.c

```diff
@@ -144,6 +144,7 @@ static p384_limb_t p384_felem_nz(const p384_limb_t in1[P384_NLIMBS]) {
 #endif // P384_USE_S2N_BIGNUM_FIELD_ARITH
 
 #define P384_FELEM_BYTES (48)
+#define P384_FELEM_WORDS ((P384_FELEM_BYTES + BN_BYTES - 1) / BN_BYTES)
```

Comment:
in the low level EC implementations (for all curves) we are making an assumption on the real size of `EC_FELEM` for a specific curve. For example, for P-384 we assume that `EC_FELEM` associated with an `EC_GROUP` for P-384 is 48 bytes long (line 146). I have two suggestions:
- rename `P384_FELEM_BYTES/WORDS` to `P384_EC_FELEM_BYTES/WORDS` (same for other curves).
- remove the asserts you added in the latest change and add a unit test instead that checks `P384_EC_FELEM_BYTES/WORDS` are equal to the real size of a `EC_FELEM` object instantiated with the corresponding `EC_GROUP`.
what do you think?